### PR TITLE
Fix/tdr 22/remove tao proctoring dependency

### PR DESCRIPTION
**Pull request by hutnikau**
Related task: https://oat-sa.atlassian.net/browse/TDR-22
see: https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/244

### CODE REVIEW

File: model/LtiLaunchDataService.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+use oat\taoLti\models\classes\LtiLaunchData;
+use oat\taoProctoring\model\deliveryLog\DeliveryLog;
+use RuntimeException;
+use oat\ltiDeliveryProvider\model\LtiLaunchDataService as BaseLtiLaunchDataService;
+
+/**
+ * Class LtiLaunchDataService
+ * @package oat\ltiProctoring\model
+ */
```
Comment (SergiiTao) [inline model/LtiLaunchDataService.php:32]: i don't think we need that

File: model/LtiLaunchDataService.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+class LtiLaunchDataService extends BaseLtiLaunchDataService
+{
+    /**
+     * @param string $deliveryExecutionId
+     * @return LtiLaunchData
+     */
+    public function findLaunchDataByDeliveryExecutionId(string $deliveryExecutionId)
```
Comment (SergiiTao) [inline model/LtiLaunchDataService.php:39]: can we add a return type here and below?

File: model/LtiLaunchDataService.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017  (original work) Open Assessment Technologies SA;
```
Comment (SergiiTao) [inline model/LtiLaunchDataService.php:18]: 2020

File: model/LtiLaunchDataService.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?php
+
```
Comment (SergiiTao) [inline model/LtiLaunchDataService.php:2]:
```suggestion
declare(strict_types=1);
```
here and below

File: model/LtiResultCustomFieldsService.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+use oat\taoDelivery\model\execution\DeliveryExecutionInterface;
+use oat\taoOutcomeUi\model\search\ResultCustomFieldsService;
+use oat\taoProctoring\model\deliveryLog\DeliveryLog;
+
+/**
+ * Class LtiResultCustomFieldsService
+ * @package oat\ltiProctoring\model
+ */
+class LtiResultCustomFieldsService extends ResultCustomFieldsService
```
Comment (SergiiTao) [inline model/LtiResultCustomFieldsService.php:33]: no tests found :)

Comment (hutnikau) [reply model/LtiResultCustomFieldsService.php:33]: I just moved this method, not created or changed it

Comment (hutnikau) [reply model/LtiResultCustomFieldsService.php:33]: Added a unit test

**Pull request merged**